pkg/node/sendoperation/serializeaddress/serializeaddress.go

```diff
@@ -29,3 +31,30 @@ func SerializeAddress(addr string) ([]byte, error) {
 
 	return append(result, addressBytes...), nil
 }
+
+func DeserializeAddress(versionedAddress []byte) (string, error) {
+	const minAddressLength = 2
```

Comment:
from the discord conversation its seems min length is  34 . isn't it ?

Reply:
yes